fixtures/new-style-code-review.js

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable */
+
+export default {
```

Comment:
Test!

![inline image with alt attr](https://gifs.bjacobel.com/internallyscreaming.gif)

[Link to same image](https://gifs.bjacobel.com/internallyscreaming.gif)

![](https://gifs.bjacobel.com/internallyscreaming.gif)

[Another link to same image](https://gifs.bjacobel.com/internallyscreaming.gif)